envconfig/config.go

```diff
@@ -220,7 +220,7 @@ var (
 	CudaVisibleDevices    = String("CUDA_VISIBLE_DEVICES")
 	HipVisibleDevices     = String("HIP_VISIBLE_DEVICES")
 	RocrVisibleDevices    = String("ROCR_VISIBLE_DEVICES")
-	VkVisibleDevices      = String("GGML_VK_VISIBLE_DEVICES")
+	VkVisibleDevices      = String("OLLAMA_VK_VISIBLE_DEVICES")
```

Comment:
I don't think `OLLAMA_VK_VISIBLE_DEVICES` exists any more but maybe we still want to log `GGML_VK_VISIBLE_DEVICES` in case the user passes it?